football/spec/models/user_spec.rb

```diff
@@ -2,15 +2,32 @@
 
 RSpec.describe :User, type: :model do
 
-  describe "validate email" do 
-    before(:each) do
+  before(:all) do
       @user = FactoryGirl.build(:user)
     end
-    subject { @user }
+  subject { @user }
+
+  describe "validate email" do 
 
     it { @user.should_not be_nil }
   	it { should validate_presence_of :email}
 
   end
 
+  describe "validate smartphone" do 
+    it { should validate_numericality_of(:smartphone).only_integer }
+  end
+
+  describe "validate name" do
+    it { should validate_presence_of :name }
+    it { should validate_length_of :name }
+  end
+
+  describe "validate password" do
+    it { should validate_presence_of :password }
+    it { should validate_confirmation_of :password }
+    it { should validate_length_of :password }
+    # it { should have_secure_password }   ********** chua lam dc 
```

Comment:
You can use [this gem](https://github.com/thoughtbot/shoulda-matchers) while you write rspec 🌵 .
Using `is_expected` instead of `should` 👍 

Btw, follow [ruby style guide](https://github.com/bbatsov/ruby-style-guide)